src/app/page.tsx

```diff
@@ -1,17 +1,27 @@
-import Image from 'next/image';
 import { Header } from '@/components/Header';
 import { PromoBanner } from '@/components/PromoBanner';
 import { barbsershopData } from '@/components/PromoBanner/PromoData';
-import Logo from '../assets/logo/logo.svg';
 
 export default function Home() {
   return (
-    <main className="overflow-hidden flex min-h-screen flex-col p-24 bg-header-bg">
-      <div className="flex">
-        <Image src={Logo} alt="Logo" width={100} height={30} className="mt-12" />
+    <main className="overflow-hidden flex md:container md:max-w-screen-2xl mx-auto flex-col bg-header-bg">
+      <div
+        className="flex flex-col justify-between bg-[url('/images/bg/hero.jpg')] bg-no-repeat bg-[left_-3.4rem_bottom_0.4rem]
+        relative
+        bg-cover
+        after:bg-[url('/images/bg/white-grunge.svg')]
+        after:z-10
+        after:h-32
+        after:bg-repeat-x
+        after:bg-[top_0.125rem_center]
```

Comment:
I would change that to `....5rem_center` because the main page will look better

Reply:
changed to 5rem. Also fixed the extra space below the background image, bg can be shrunk now. The problem was occurring due to the transform:scale(). Now it looks much better.